### ISSUE TITLE
loss always drop to around 0.36

### ISSUE THREAD

**Issue by hanmaokun** [llens/CryptoCurrencyTrader#28]
I tried to train, with no code modification, the train loss alwasy drops to around 0.36, despite lr change. 
And one more issue, I am confused with the line 'portfolio_value_temp = portfolio_array * shrinking_factor * price_array' in porftfolio_functions.py, could you explain what's that mean? thx!

**Comment by llens** [comment]
The model was set up to load a pre-trained model.h5 file, I have deleted that line now, so it should start training from scratch now. This line calculates the portfolio value after combining the portfolio vector allocations, the price of asset and the shrinking factor which is the reduction in value due to transaction fees. The mathematics of this is outlined in the paper linked in the README file.

**Comment by llens** [comment]
"This line calculates ..." should be "The line you quote ..."

**Comment by hanmaokun** [comment]
thx, I just noticed I need went back to the paper you mentioned :)
